gcsfilesystem/src/test/scala/cromwell/gcsfilesystem/GoogleConfigurationSpec.scala

```diff
@@ -0,0 +1,161 @@
+package cromwell.gcsfilesystem
```

Comment:
Not sure what you mean?  I didn't think there were config changes in this PR.  Lots of them in the one I'm working on though...